src/posit/connect/vanities.py

```diff
@@ -71,10 +71,6 @@ def __init__(
         self._after_destroy = after_destroy
         self._content_guid = kwargs["content_guid"]
 
-    @property
-    def _endpoint(self):
-        return self.params.url + f"v1/content/{self._content_guid}/vanity"
-
```

Comment:
Just clarifying my comment from earlier...

I'm not against the helper function `def _endpoint(self):` , I just don't want it as a `@property`.

Reply:
Ah, I see the distinction you are making. Thanks for the clarification. Either way, this will go away when migrating this module to use `Active`.